### PR TITLE
nw-cluster-mtu-change: generate MachineConfigs with Butane

### PR DESCRIPTION
**Pull request by bgilbert**
Version(s):
4.10+

Issue:
n/a

Link to docs preview:
https://deploy-preview-45749--osdocs.netlify.app/openshift-enterprise/latest/networking/changing-cluster-network-mtu.html

Additional information:
In recent OpenShift versions, we generally recommend Butane for preprocessing configs, rather than manually base64-encoding files and inserting them inline.  Since the procedure involves extracting and modifying a file, use Butane's local file inclusion rather than inlining the file into the Butane config.

### CODE REVIEW

**Comment by bgilbert** [comment]
cc @jboxman-rh

File: modules/nw-cluster-mtu-change.adoc [hunk 1]
```diff
@@ -212,90 +212,56 @@ interface-name=enp1s0
...
+<2> Specify the local filename for the updated NetworkManager configuration file from the previous step.
 
-`<encoded_config>`:: Specifies the base64 encoding for the updated NetworkManager configuration.
-`<connection_name>`:: Specifies the NetworkManager connection name for the primary network interface.
---
+.... Create `MachineConfig` objects from the Butane configs:
++
+[source,terminal]
+----
+$ for manifest in control-plane-interface worker-interface; do
```
Comment (tmalove) [inline modules/nw-cluster-mtu-change.adoc:261]: Not sure if the semicolon is a part of the command...should it be a period instead?

Comment (bgilbert) [reply modules/nw-cluster-mtu-change.adoc:261]: No, the semicolon is correct.  It separates multiple commands on the same line; in this case, `for` and `do`.

File: modules/nw-cluster-mtu-change.adoc [hunk 2]
```diff
@@ -212,90 +212,56 @@ interface-name=enp1s0
...
-+
---
-where:
+<1> Specify the NetworkManager connection name for the primary network interface.
+<2> Specify the local filename for the updated NetworkManager configuration file from the previous step.
 
-`<encoded_config>`:: Specifies the base64 encoding for the updated NetworkManager configuration.
-`<connection_name>`:: Specifies the NetworkManager connection name for the primary network interface.
---
+.... Create `MachineConfig` objects from the Butane configs:
```
Comment (jeana-redhat) [inline modules/nw-cluster-mtu-change.adoc:257]:
```suggestion
.... Create `MachineConfig` objects from the Butane configs by running the following command:
```

**Review by jeana-redhat — COMMENTED** [review]
Just one small style guide thing (from [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#code-blocks-command-syntax-and-example-output), for reference :slightly_smiling_face:)

**Pull request merged**